### PR TITLE
Native YAML - cloud/softlayer/sl_vm.py

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
cloud/softlayer/*

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by Fale** [comment]
Migrated from https://github.com/ansible/ansible-modules-extras/pull/3629

**Comment by gundalow** [comment]
@mcltn Does this look OK to you?

File: lib/ansible/modules/cloud/softlayer/sl_vm.py [hunk 1]
```diff
@@ -149,8 +149,7 @@
   gather_facts: False
   tasks:
   - name: Build instance request
-    local_action:
```
Comment (gundalow) [inline lib/ansible/modules/cloud/softlayer/sl_vm.py:152]: I don't see delegate_to.
Should this be ran local or remote?

File: lib/ansible/modules/cloud/softlayer/sl_vm.py [hunk 2]
```diff
@@ -170,8 +169,7 @@
   gather_facts: False
   tasks:
   - name: Build instances request
-    local_action:
```
Comment (gundalow) [inline lib/ansible/modules/cloud/softlayer/sl_vm.py:173]: I don't see delegate_to.
Should this be ran local or remote?

**Review by gundalow — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/softlayer/sl_vm.py (hunk 1 above)
Comment (Fale) [reply lib/ansible/modules/cloud/softlayer/sl_vm.py:152]: That's remote, running against localhost (which makes the action to be performed... on localhost)

File: lib/ansible/modules/cloud/softlayer/sl_vm.py (hunk 2 above)
Comment (Fale) [reply lib/ansible/modules/cloud/softlayer/sl_vm.py:173]: That's remote, running against localhost (which makes the action to be performed... on localhost)

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
I'll give the maintainer a bit of time to look at this and merge at the next meeting if I don't hear anything

**Comment by mcltn** [comment]
This looks good to me. Thanks! "shipit"

**Comment by gundalow** [comment]
@mcltn Thanks for the quick review, merging

**Pull request merged**